bin/node/runtime/src/lib.rs

```diff
@@ -746,7 +747,9 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type MinerConfig = Self;
 	type SignedMaxSubmissions = ConstU32<10>;
 	type SignedRewardBase = SignedRewardBase;
-	type SignedDepositBase = SignedDepositBase;
+	type SignedDepositBase = GeometricDepositBase<Runtime>;
+	type SignedFixedDepositBase = SignedDepositBase;
```

Comment:
Is `SignedFixedDepositBase` used anymore? Can we get rid of it? I believe if we want to use fixed deposit base, we should provide `SignedDepositBase ` with a `FixedDepositBase<>` instead of `GeometricDepositBase<>`?